doc/api/api_changes/2018-01_TAC.rst

```diff
@@ -0,0 +1,8 @@
+
+Simplify String Categorical handling
+------------------------------------
+
+ - Only handling `str` (not `bytes` in pyhon3).  This disallows using integers or
```

Comment:
sp: `python3`

(sorry, I know we're supposed to push minor editorial changes directly, but I'm away from my main machine ATM)